src/esp/assets/ResourceManager.cpp

```diff
@@ -1833,7 +1899,15 @@ bool ResourceManager::loadSUNCGHouseFile(const AssetInfo& houseInfo,
         nodeIds.push_back(id);
         objectNode.setId(nodeIndex);
         if (info.type == AssetType::SUNCG_OBJECT) {
-          loadGeneralMeshData(info, &objectNode, drawables);
+          CHECK(loadRenderAsset(info));
```

Comment:
This codepath is untested so who knows if it actually works. https://cvmlp.slack.com/archives/G0131KVLBLL/p1604940887000900